TwinePlatform/twine-monolith
# 💬 volunteer app copy changes

fixes #278 
fixes #279 

### Changes
- Move projects input above activities input
- Address changes mentioned #278

### Testing Requirements
  - Web version
	- run `ionic serve`
    - Observe login page has placeholders removed and input boxes enlarged
    - Login as `VOLUNTEER`
    - Observe dashboard has copy changed as per recommendations in #278 
    - Log out
    - Observe Twine logo has been removed from register page
    - Log in as `VOLUNTEER_ADMIN`
    - Observe nav bar has the the copy change as per recommendations in #278

### Release Requirements
- Wait for changes related to #280, #281, #282 to be merged
- Build app 
- release internal testing version (Android & Test Flight) (make available to PO)
- Release to app stores on PO approval

### Manual Deployment
- Volunteer app
